geomesa-index-api/src/test/scala/org/locationtech/geomesa/index/geotools/GeoMesaDataStoreTest.scala

```diff
@@ -199,7 +123,7 @@ class GeoMesaDataStoreTest extends Specification {
       results must haveLength(10)
     }
     "block queries which would cause a full table scan" in {
-      val sft = SimpleFeatureTypes.createType("test",
+      val sft = SimpleFeatureTypes.createType("61b44359ddb84822983587389d6a28a4",
```

Comment:
Catchy type name!

Reply:
haha, i made it `UUID.randomUUID` but then had to replace it with a constant value so that the typesafe conf file lined up :)